site/docs/guide/introduction.md

```diff
@@ -129,9 +130,46 @@ Please choose what you think is best for you.
 
 ### Prerequisites for Deno
 
-[Install Deno](https://deno.land/#installation) if you have not done that already.
-When you have created your bot, for example in a file called `bot.ts`, you can run it via `deno run --allow-net bot.ts`.
-You can stop it again with `Ctrl+C`.
+Before you can start creating your bot, let's first spend a few minutes on a proper setup for software development.
+This means installing a few tools.
+
+#### Preparing Your Machine for Development
+
+[Install Deno](https://deno.land/manual/getting_started/installation#download-and-install) if you have not done it already.
+
+You also need a text editor that fits well with coding.
+The best one for Deno is Visual Studio Code.
+[Install it.](https://code.visualstudio.com/)
+
+Next, you need to connect VSCode and Deno.
+That's very simple: There is an extension for VSCode that does everything automatically.
+You can install it [as described here](https://marketplace.visualstudio.com/items?itemName=denoland.vscode-deno).
+
+Your system is now ready for bot development! :tada:
+
+#### Developing a Bot
+
+Create a new directory somewhere.
+It will contain your bot project.
+Open this new directory in VSCode.
+
+```sh
+mkdir ./my-bot
+cd ./my-bot
+code .
+```
+
+In VSCode, turn this empty directory into a Deno project.
+Hit `Ctrl+Shift+P`, type "deno init" and hit Enter.
+The bottom right of your editor should now display the Deno version on your system.
+
+Your Deno project is now ready so you can create your bot.
```

Comment:
```suggestion
Your Deno development environment should now be ready. So, you can now start writing your bot.
```

Reply:
```suggestion
Your Deno development environment is ready.
You can now create your bot.
```
?

Reply:
Better. But I still prefer write instead of create.

Reply:
I missed that part

Reply:
```suggestion
Your Deno development environment is ready.
You can now start writing your bot.
```
?